packages/web/graphql/fragments.ts

```diff
@@ -10,25 +10,25 @@ export const PlayerFragment = gql`
     ethereum_address
     availability_hours
     timezone
-    ColorAspect {
+    color_aspect {
```

Comment:
References in:

seed-local-db.js
graphql/getPersonalityInfo.ts

Reply:
In getPersonalityInfo it is still loading all the ColorAspects as opposed to that of the current player